comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -23,38 +24,100 @@ public static bool IsDirectory(string path)
 
 		// Return the total size, in bytes, of all the files below the given directory
 		public static long GetTotalSize(string directory)
-		{
-			throw new NotImplementedException();
-		}
+        {
+            string[] files;
+            long fileSize = 0;
+
+            files = Directory.GetFiles(directory, "*.*", SearchOption.AllDirectories);
```

Comment:
Nice! Not the solution I was looking for but good use of the .NET standard library functions